x11-misc/xkeyboard-config/xkeyboard-config-2.39.ebuild

```diff
@@ -19,21 +21,33 @@ fi
 
 LICENSE="MIT"
 SLOT="0"
+IUSE="man"
 
 DEPEND=""
 RDEPEND=""
 BDEPEND="
 	${PYTHON_DEPS}
 	dev-lang/perl
-	dev-libs/libxslt
+	man? ( dev-libs/libxslt )
 	sys-devel/gettext
 "
 
+src_prepare() {
+	default
+	plocale_get_locales > po/LINGUAS || die
+}
+
 pkg_setup() {
 	python-any-r1_pkg_setup
 }
 
 src_configure() {
+	if ! use man; then
+		# Make sure xslt is disabled
+		sed -i -e 's/if xsltproc\.found()/if false/' 'meson.build'
+	fi
+	# Make sure pytest is disabled
```

Comment:
(Also, the comment is a bit "# foo the bar", the more interesting part is why it's okay - and why we want to - disable pytest. But as per my other comment, it looks like we may not want to do that.)